store/external_approval.go

```diff
@@ -147,6 +164,25 @@ func (s *Store) findExternalApprovalRaw(ctx context.Context, find *api.ExternalA
 	return rawList, nil
 }
 
+func (s *Store) getExternalApprovalRawByIssueID(ctx context.Context, issueID int) (*externalApprovalRaw, error) {
```

Comment:
We can avoid this function as long as we use GetExternalApprovalByIssueID to call FindExternalApproval.